chef/cookbooks/hyperv/recipes/windows_features.rb

```diff
@@ -8,27 +8,31 @@
     node.set[:windows_features_installed] = true
```

Comment:
I understand, but then why do we even set the attribute? If we set it, then it should have the same type as below